packages/pluginutils/package.json

```diff
@@ -75,12 +75,17 @@
       "!**/types.ts"
     ]
   },
-  "module": "dist/index.es.js",
+  "exports": {
+    "require": "./dist/cjs/index.js",
+    "import": "./dist/es/index.js"
+  },
+  "module": "./dist/es/index.js",
```

Comment:
question about the choice to nest these within a directory: our prior file format was `index.[format].[extension]` for output files. e.g. `index.es.js`. what's the reasoning behind the choice to put them into directories?

Reply:
es/package.json with {type:module}

Reply:
As an alternative to .mjs extension.

Reply:
OK. Can you explain in the PR description why we need mjs alternative? I've added a bunch more people to review. We're going to need to get majority buy-in to move forward with this since it's a pretty big paradigm shift.